ui/admin/app/components/roles/role/actions/index.hbs

```diff
@@ -3,38 +3,21 @@
   SPDX-License-Identifier: BUSL-1.1
 }}
 
-{{!TODO Refactor page to use hds dropdown in ICU-13879}}
-<Rose::Dropdown
-  @text={{t 'actions.manage'}}
-  @dropdownRight={{true}}
-  as |dropdown|
->
+<Hds::Dropdown as |dd|>
+  <dd.ToggleButton @text= {{t 'actions.manage'}} @color ='secondary'/>
   {{#if (can 'addPrincipals role' @model)}}
-    <dropdown.link
-      @route='scopes.scope.roles.role.add-principals'
-      data-test-manage-dropdown-principals
```

Comment:
so far looking really good! 
I recommend adding back in this data-test selector and the selector on line 24 since they are used in tests.  Here is a link to an [example](https://github.com/mainmatter/ember-test-selectors?tab=readme-ov-file#usage) of how they are used in tests.